packages/n4s/src/rules/__tests__/isValueOf.test.ts

```diff
@@ -0,0 +1,36 @@
+import { enforce } from 'enforce';
+import { isValueOf, isNotValueOf } from 'isValueOf';
+
+const objectToTest = {
+  a: 'Alpha',
+  b: 'Bravo',
+  c: 'Charlie',
+};
+
+describe('isValueOf tests', () => {
+  describe('When "Bravo" is the value of key "b"', () => {
+    it('Should return true using enforce', () => {
+      enforce('Bravo').isValueOf(objectToTest);
+    });
+    it('Should return true', () => {
+      expect(isValueOf('Bravo', objectToTest)).toBe(true);
+    });
+  });
+  describe('When "Delta" is not a value of any key', () => {
+    expect(isValueOf('Delta', objectToTest)).toBe(false);
+  });
```

Comment:
You're missing the `it` statement. This means that there's no test run here. Same on line 34.

```suggestion
  describe('When "Delta" is not a value of any key', () => {
    expect(isValueOf('Delta', objectToTest)).toBe(false);
  });
```

Reply:
fixed wrong structured tests + added more tests